src/components/features/Promotions/Promotions.module.css

```diff
@@ -0,0 +1,139 @@
+@charset "UTF-8";
+.root {
+  padding: 5rem 0;
+  /* tekst i overlay dużego zdjęcia po lewej stronie komponentu */
+  /* mniejsze zdjęcie po prawej od góry; tekst podzielony - osobne style dla każdego słowa, według grafiki projektu  */
+  /* mniejsze zdjęcie po prawej od dołu; tekst podzielony - osobne style dla text_2 i spanu, według grafiki projektu */
+  /* style dla każdego zdjęcia w komponencie */
+}
+
+.root .bigLeft {
+  position: relative;
+  height: 500px;
+}
+
+.root .bigLeft .overlay {
+  position: absolute;
+  background-color: #d8d6d6;
+  opacity: 60%;
+  height: 90%;
+  width: 95%;
+  top: 50%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
+          transform: translate(-50%, -50%);
+}
+
+.root .bigLeft .opacityText {
+  position: absolute;
+  text-align: left;
```

Comment:
Do we really need all this absolute positioning for text content? I bet you can do well without :) Remember: use absolute positioning only if there is no other way. Here we have background-image, white overlay, and then text, so we have to put overlay or text on top - that's true. But still, we can choose, which element should be positioned absolutely. Do we have to use `position: absolute` for text? No! Use it for overlay and it will really help with the RWD experience :)